internal/pkg/instrumentation/bpf/database/sql/probe.go

```diff
@@ -97,6 +104,23 @@ func processFn(e *event) ptrace.SpanSlice {
 		span.Attributes().PutStr(string(semconv.DBQueryTextKey), query)
 	}
 
+	includeOperationVal := os.Getenv(IncludeDBOperationEnvVar)
+	if includeOperationVal != "" {
+		include, err := strconv.ParseBool(includeOperationVal)
```

Comment:
We could read the env var once and cache the result ( not sure how expensive it is).

Reply:
`Getenv` ultimately calls `syscall.Getenv`: https://cs.opensource.google/go/go/+/refs/tags/go1.23.4:src/syscall/env_unix.go;l=69-89

This is pretty well cached and handles updates concurrently and efficiently. We should be fine making this call each time.